src/Resolvers/Microsoft.NET.Sdk.WorkloadMSBuildSdkResolver/Microsoft.NET.Sdk.WorkloadMSBuildSdkResolver.csproj

```diff
@@ -45,6 +45,7 @@
     <ItemGroup>
       <ManifestLines Include="&lt;SdkResolver&gt;" />
       <ManifestLines Include="&lt;Path&gt;$(_TargetPathRelativePath)&lt;/Path&gt;" />
+      <ManifestLines Include="&lt;ResolvableSdkPattern&gt;Microsoft\..*|Samsung\..*|GtkSharp\..*&lt;/ResolvableSdkPattern&gt;" />
```

Comment:
This is the content from the xml, just replicated in this 'generation' step for the .NET framework version of this resolver.